metals/src/main/scala/scala/meta/metals/Configuration.scala

```diff
@@ -30,6 +31,7 @@ object Configuration {
   )
   @JsonCodec case class Scalac(enabled: Boolean = false)
   @JsonCodec case class Hover(enabled: Boolean = false)
```

Comment:
Unrelated to this PR. I think it might be safe to enable hover by default. It should work as robustly if more robustly than goto definition